daphne_server/src/router/aggregator.rs

```diff
@@ -107,4 +148,81 @@ mod test {
 
         assert_eq!(status, StatusCode::OK);
     }
+
+    // Check that a signature produced by Daphne-Worker will be verified properly by the Clients.
+    #[test]
+    fn rondtrip_sign_hpke_config() {
+        fn os<S: AsRef<OsStr> + ?Sized>(s: &S) -> &OsStr {
+            OsStr::new(s)
+        }
+
+        let mut signing_key_file = tempfile::NamedTempFile::new().unwrap();
+        let status = Command::new("openssl")
+            .args([
+                os("ecparam"),
+                os("-name"),
+                os("prime256v1"),
+                os("-genkey"),
+                os("-noout"),
+                os("-out"),
+                os(signing_key_file.path()),
+            ])
+            .spawn()
+            .unwrap()
+            .wait()
+            .unwrap();
+
+        assert!(status.success(), "{status:?}");
+
+        let mut cert_file = tempfile::NamedTempFile::new().unwrap();
+
+        let status = Command::new("openssl")
+            .args([
+                os("req"),
+                os("-key"),
+                os(signing_key_file.path()),
+                os("-new"),
+                os("-x509"),
+                os("-days"),
+                os("1"),
+                os("-out"),
+                os(cert_file.path()),
+                os("-subj"),
+                os("/C=PT/L=Braga/O=Cloudflare Lda/CN=cloudflare.com"),
+            ])
+            .spawn()
+            .unwrap()
+            .wait()
+            .unwrap();
+
+        assert!(status.success(), "{status:?}");
+
+        const PAYLOAD: &[u8] = b"dummy HPKE configuration";
+        let signing_key = SigningKey::from({
+            let mut buf = String::new();
+            signing_key_file.read_to_string(&mut buf).unwrap();
+            p256::SecretKey::from_sec1_pem(&buf).unwrap()
+        });
+
+        let resp = DapResponse {
+            version: DapVersion::Draft02,
```

Comment:
Just to give us a little less work when it's time to rip out draft 02.
```suggestion
            version: DapVersion::Draft09,
```